### PR TITLE
adding swagger docs for election and elector endpoints

### PR DESCRIPTION
**Pull request by aditya-cherukuri**
Just the 1st phase of creating docs to see if I'm doing this right and I'm not missing anything.

Note: When you pull this code locally you would have to run 'npm run dev' so the webpack mix scripts copy the assets required to display the docs. If required I can remove this and instead commit the assets to the public/swagger directory.

### CODE REVIEW

File: app/Definitions/Election.php [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+}
+
+/**
+ * @SWG\Definition(
+ *     definition="ElectionWithCreator",
+ *     type="object",
+ *     @SWG\Xml(name="ElectionWithCreator")
+ * )
+ */
+class ElectionWithCreator
```
Comment (andrewshell) [inline app/Definitions/Election.php]: We're following PSR-2 coding conventions so each class should be in its own file.

File: webpack.mix.js [hunk 2]
```diff
@@ -13,3 +13,8 @@ const { mix } = require('laravel-mix');
 
 mix.js('resources/assets/js/app.js', 'public/js')
    .sass('resources/assets/sass/app.scss', 'public/css');
+
+
+mix.copy('vendor/swagger-api/swagger-ui/dist/swagger-ui.css', 'public/swagger/asset/swagger-ui.css');
```
Comment (andrewshell) [inline webpack.mix.js]: It shouldn't be needed to copy these files over. They are automatically handled by l5-swagger

Comment (aditya-cherukuri) [reply webpack.mix.js]: It seems like the routes are registered. But when I remove the assets I get 404's on the requests that try to fetch the assets. I'll try debugging more and see if I can find the root cause

Comment (aditya-cherukuri) [reply webpack.mix.js]: https://github.com/DarkaOnLine/L5-Swagger/issues/75 references the issue I had and the solution has also been posted. I had to remove the lines from my nginx config. I'm going to remove the webpack code that I wrote.

**Pull request merged**